@pauliescanlon/gatsby-mdx-embed/src/components/Twitch/Twitch.tsx

```diff
@@ -32,7 +38,7 @@ export const Twitch: FunctionComponent<ITwitchProps> = ({
     >
       <iframe
         title={`twitch-${twitchId}`}
-        src={`https://player.twitch.tv/?autoplay=${autoPlay}&t=${h}h${m}m${s}s&video=v${twitchId}`}
+        src={`https://player.twitch.tv/?autoplay=${autoPlay}&t=${h}h${m}m${s}s&video=v${twitchId}&channel=${channel}&collection=${collection}`}
```

Comment:
Hey @zslabs thanks for the contribution, Is it possible to add an example story in the mdx so show what the `twitchId` url might look like if  a channel or collection was included

Reply:
👋 I've updated the examples (and added a "channel" one) to utilize the required `parent` param that needs to be passed through to each instance. If you run this locally, you'll need to change these examples to `localhost` to be able to see them. I honestly couldn't find a collection 🤷 but if you have an example one I'm happy to update the PR to include it.